CHANGES.md

```diff
@@ -0,0 +1,8 @@
+## dev (Unreleased)
+
+## v0.1.0 (August 2021)
+
+- First release (`0.1.0`) by
```

Comment:
XHTMLBoy should ne mentioned no?

Reply:
You're right, I have written a more expansive Changelog: https://github.com/xvw/preface/blob/f591f4461de845b9243e5558d13141de9e009ab7/CHANGES.md

Reply:
Perfect